### ISSUE TITLE
Fixture has dependency on steal

### ISSUE THREAD

**Issue by webbower** [jupiterjs/jquerymx#62]
https://github.com/jupiterjs/jquerymx/blob/master/dom/fixture/fixture.js#L35

Fixture can no longer be used on its own.

**Comment by justinbmeyer** [comment]
It can, just don't have any paths that start with //.

**Comment by webbower** [comment]
I double-checked and found I didn't... explicitly. I figured out what's triggering the error I'm getting. For some reason, it's dynamically creating a fixture URL that looks like this:

//<ns>/<classname>/fixtures/<classname>s.json (The Model class was called "NB.Stations" and the URL it was choking on, that I never explicitly wrote ANYWHERE in my code, was "//nb/stations/fixtures/stationss.json".)

I noticed that the fixture URL had two S's at the end so I tried making the Model class name singular (NB.Station) and it stopped throwing the error. Any idea why that is? Maybe a bug?

Matt Bower
matt@webbower.com
www.webbower.com
p: 530-591-4357
